detd/service.py

```diff
@@ -250,13 +249,12 @@ def _add_talker(self, request):
         size = request.size
         interface_name = request.interface
 
-        options = Options()
-        options.qdiscmap = request.qdiscmap
+        hints = Hints()
```

Comment:
The service should get the hints through the request. For that we need to add the hints details to the IPC protobuf IDL, and also on the proxy side at the other end of the UDS.

Then here we could simply initialize Hints as the other objects (Interface, StreamConfiguration, TrafficSpecification, etc) are initialized below.